test/utils/noderesourcetopologies/noderesourcetopologies.go

```diff
@@ -256,6 +256,16 @@ func ZoneResourcesMatchesRequest(resources []nrtv1alpha1.ResourceInfo, requests
 	return true
 }
 
+func FilterSingleNumaNodePolicyNrts(list []nrtv1alpha1.NodeResourceTopology) []nrtv1alpha1.NodeResourceTopology {
```

Comment:
it's ok for now, but I'd have extended `FilterTopologyManagerPolicy` to take a set of policy into account.

Reply:
Refactored.